include/fmt/format.h

```diff
@@ -265,6 +265,37 @@ FMT_END_NAMESPACE
 
 FMT_BEGIN_NAMESPACE
 namespace detail {
+
+template <typename Char, typename Streambuf, typename Streamsize>
```

Comment:
Both `Char` and `Streamsize` can be determined from `Streambuf`: `Char` is `Streambuf::char_type` and `Streamsize` is the return type of `xsputn`.